File: common/src/main/java/cn/breadnicecat/candycraftce/block/blocks/SweetGrassBlock.java
package cn.breadnicecat.candycraftce.block.blocks;

import net.minecraft.Util;
import org.jetbrains.annotations.NotNull;

import static cn.breadnicecat.candycraftce.utils.ResourceUtils.prefix;

/**
 * Created in 2024/2/14 21:38
 * Project: candycraftce
 *
 * @author <a href="https://github.com/BreadNiceCat">Bread_NiceCat</a>
 * <p>
 */
public class SweetGrassBlock extends CandyPlantBlock {
	public static final String DESCRIPTION_ID = Util.makeDescriptionId("block", prefix("sweet_grass"));

	public SweetGrassBlock(Properties properties) {
		super(properties);
	}

	@Override
	public @NotNull String getDescriptionId() {
		return DESCRIPTION_ID;
	}
}

File: src/main/kotlin/cn/breadnicecat/candycraftce/core/block/blocks/plant/SweetGrassBlock.kt
package cn.breadnicecat.candycraftce.core.block.blocks.plant

import cn.breadnicecat.candycraftce.utils.CUtils.modLoc
import net.minecraft.Util

/**
 * Created by NiceCat on 2026/2/28.
 * Project: candycraftce
 * @author <a href="https://github.com/BreadNiceCat">Bread_NiceCat</a>
 *
 */
class SweetGrassBlock(properties: Properties) : CandyPlantBlock(properties) {
    companion object {
        val descId: String = Util.makeDescriptionId("block", "sweet_grass".modLoc())
    }

    override fun getDescriptionId() = descId
}